activerecord/lib/active_record/connection_adapters/abstract/query_cache.rb

```diff
@@ -169,8 +177,10 @@ def enable_query_cache!
       end
 
       # Disable the query cache within the block.
-      def uncached(&)
-        pool.disable_query_cache(&)
+      #
+      # Setting <tt>dirties: false</tt> will prevent the query cache from being cleared by write operations.
```

Comment:
```suggestion
      # Set <tt>dirties: false</tt> to prevent query caches on all connections from being cleared by write operations.
      # (By default, write operations dirty all connections' query caches in case they are replicas whose cache would now be outdated.)
```